### PR TITLE
[WIP] E2E test for create integration command

### PR DESCRIPTION
**Pull request by dimitraz**
**Describe what this PR does and why we need it**:
Integration test for the `create integration` command
The test is focused on the binding between `fh-sync-server` and `keycloak`.

**Changes proposed in this pull request**
- Add integration test and golden files for the `create integration` command

**This PR fixes the following issue**
[AEROGEAR-1925](https://issues.jboss.org/browse/AEROGEAR-1925)

### CODE REVIEW

File: integration/create_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	args := []string{"create", "serviceinstance", si.Name, si.Namespace}
+	args = append(args, si.Params...)
+	cmd := exec.Command(*executable, args...)
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to create service instance %s: %v", si.Name, err)
+	}
+
+	fmt.Println(string(output))
```
Comment (JameelB) [inline integration/create_integration_test.go]: Is this println needed?

**Comment by maleck13** [comment]
@AdamSaleh this is the kind of test where I think we want to add a label to the PR to run it rather than run it every time

@dimitraz This is looking great!! Will give it a good look over on Monday

File: integration/create_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+func createInstance(t *testing.T, si *ProvisionServiceParams) {
+	args := []string{"create", "serviceinstance", si.ServiceName, si.Namespace}
+	args = append(args, si.Params...)
+	cmd := exec.Command(*executable, args...)
+
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to create service instance %s: %v", si.ServiceName, err)
```
Comment (JameelB) [inline integration/create_integration_test.go]: The `err` doesn't seem to be giving much information on what the actual error is. It seems to just be something along the lines of `exit status 1`. I suggest printing out the `output` here as it seems that this contains the actual error message

Comment (dimitraz) [reply integration/create_integration_test.go]: Yeah i noticed that too. I'll give it a go, thanks!

File: integration/create_integration_test.go (hunk 1 above)
Comment (dimitraz) [reply integration/create_integration_test.go]: Not completely, but i kept it in because it gives a vague idea of how long the provisioning is taking

**Comment by philbrookes** [comment]
Retest this please.

**Pull request merged**